frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -67,7 +67,11 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
 
       <Flex direction={{ default: 'column' }} grow={{ default: 'grow' }}>
         {links?.length > 0 ? (
-          <SimpleList isControlled={false} className="ocs-getting-started-card__list">
+          <SimpleList
+            isControlled={false}
+            className="ocs-getting-started-card__list"
+            data-test="getting-started-list"
+          >
```

Comment:
I would expect that you don't need an additional `data-test` attribute here because the card has already an data-test it ([here](https://github.com/openshift/console/blob/16dbeeebad1e334f456f20a45d50b34e1f8374b6/frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx#L55)) and the individual items [here](https://github.com/openshift/console/blob/16dbeeebad1e334f456f20a45d50b34e1f8374b6/frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx#L91-L95)

Reply:
The item in the list can keep on changing so I've used this so as to get the first element from the list irrespective of which quick start present